while True:
    try:
        enter = float(input('Введите число '))
        print(100/enter)

    except ValueError:
        print('Вы ввели не число!!!')

    except ZeroDivisionError:
        print('Делить на ноль нельзя')

    else:
        print('Пользователь молодец, с первого раза')

    finally:
        print('Вывод finally')

print('Всё норм')

'''
Для обработки исключений используется конструкция try - except. В блоке try мы выполняем инструкцию, 
которая может породить исключение, а в блоке except мы перехватываем их. При этом перехватываются как само исключение, 
так и его потомки.
'''